index.html

```diff
@@ -1038,56 +1350,306 @@ <h3>
                   </li>
                 </ol>
               </li>
-            </ol>
-            <dl class="switch">
-              <dt id="errors-reasons">
-                Dealing with failures:
-              </dt>
-              <dd>
-                <ul>
-                  <li>If acquiring a position fails, do one of the following
-                  based on the condition that matches the failure:
-                    <dl class="switch">
-                      <dt id="permission-denied">
-                        User or system denied permission:
+            </ol></del> <ins cite="#a12">
+            <ol>
+              <li>Let |precisePermission| be [=get the current permission
```

Comment:
When declaring a new variable with "Let", include the type name if there's an actual IDL type for it.

|precisePermission:PermissionState|